FIP process amendment to integrate spec update (#11)
filecoin-project/FIPs: FIPS/fip-0001.md

` created:` \<date created on\>

> ` spec-sec:` Spec section(s) that the FIP is updating (comma-separated if more than one).

Reviewer: ```suggestion
`spec-sections:` A list of Spec sections that the FIP is updating.

`spec-pr:` The URL for the PR updating the Spec.
```

The `created` header records the date that the FIP was assigned a number. Both headers should be in yyyy-mm-dd format, e.g. 2001-08-14.

> #### `spec-sec`

Reviewer: ```suggestion
#### `spec-sections`
```

> Spec section(s) that the FIP is updating, based on the folder structure of the spec repository. For instance, if a FIP updates the operation of the `mpool`, the entry in this field should be `<filecoin_blockchain__message_pool>`. When a FIP updates multiple sections, all of them should be listed in a comma-separated manner.

Reviewer: ```suggestion
Spec sections that the FIP is updating, each section has an ID that can be found in the Spec website under each section "three dot" menu. Example: https://github.com/filecoin-project/FIPs/pull/11#issuecomment-713435889.

Example:
```yaml 
spec-setions:
  - section-intro
  - section-libraries.libp2p
  - section-systems.filecoin_markets.storage_market.deal-flow
```

#### `spec-pr`

The URL for the PR that updates the Spec according to the current FIP.

Example:
```yaml
spec-pr: https://github.com/filecoin-project/specs/pull/1204
```

```

Reviewer: the preview for the suggested change look a bit off but the source is correct.

- Merge the corresponding pull request

> - Check whether the FIP needs to be accompanied by an update to the specification and if os, make sure that a PR to spec repository has been submitted and that the text that updates the spec is reflecting the changes that the FIP proposes.

Reviewer: ```suggestion
- Check whether the FIP needs to be accompanied by an update to the specification and if so, make sure that a PR to spec repository has been submitted and that the text that updates the spec is reflecting the changes that the FIP proposes.
```
